### PR TITLE
HOSTEDCP-1027: Add Create kubeconfig for HCP CLI

### PR DESCRIPTION
**Pull request by bryan-cox**
**What this PR does / why we need it**:
Add Create kubeconfig for HCP CLI

**Which issue(s) this PR fixes**:
Fixes [HOSTEDCP-1027](https://issues.redhat.com/browse/HOSTEDCP-1027)

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.

### CODE REVIEW

File: cmd/kubeconfig/create.go [hunk 1]
```diff
@@ -22,12 +22,12 @@ import (
...
 // TODO: NEXT: incorporate into an fzf workflow
 
-const description string = `
+const Description string = `
 This command renders kubeconfigs for HostedClusters.
 
 If a single cluster is identified, the kubeconfig is printed to stdout.
 
-If no clusters is identified, this command renders a kubeconfig with a context
+If no clusters are identified, this command renders a kubeconfig with a context
```
Comment (csrwng) [inline cmd/kubeconfig/create.go:30]: s/identified/specified/

File: cmd/kubeconfig/create.go [hunk 2]
```diff
@@ -22,12 +22,12 @@ import (
 
 // TODO: NEXT: incorporate into an fzf workflow
 
-const description string = `
+const Description string = `
 This command renders kubeconfigs for HostedClusters.
 
 If a single cluster is identified, the kubeconfig is printed to stdout.
```
Comment (csrwng) [inline cmd/kubeconfig/create.go:28]: s/identified/specified/

File: product-cli/cmd/kubeconfig/create.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		SilenceUsage: true,
+	}
+
+	opts := options{}
+
+	cmd.Flags().StringVar(&opts.namespace, "namespace", opts.namespace, "A HostedCluster namespace. Will default to 'clusters' if a --name is supplied")
+	cmd.Flags().StringVar(&opts.name, "name", opts.name, "A HostedCluster name")
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		if opts.name != "" && opts.namespace == "" {
```
Comment (csrwng) [inline product-cli/cmd/kubeconfig/create.go:33]: just initialize to "clusters" in line 27

Comment (bryan-cox) [reply product-cli/cmd/kubeconfig/create.go:33]: Should I do that for the regular version as well (cmd/kubeconfig/create.go)?

Comment (bryan-cox) [reply product-cli/cmd/kubeconfig/create.go]: I just went ahead and updated it as well.

File: product-cli/cmd/kubeconfig/create.go [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	opts := options{
+		namespace: "clusters",
+	}
+
+	cmd.Flags().StringVar(&opts.namespace, "namespace", opts.namespace, "A HostedCluster namespace. Defaults to 'clusters'.")
+	cmd.Flags().StringVar(&opts.name, "name", opts.name, "A HostedCluster name.")
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		if err := hypershiftkubeconfig.Render(cmd.Context(), opts.namespace, opts.name); err != nil {
+			_, _ = fmt.Fprintf(os.Stderr, "%s\n", err)
```
Comment (csrwng) [inline product-cli/cmd/kubeconfig/create.go:36]: Use something like "Error: %s" given that errors should all start with lowercase

**Pull request merged**